packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -361,6 +361,7 @@ export class NotifiFrontendClient {
           authorization = newAuthorization;
         }
       } catch (e: unknown) {
+        await this.logOut();
```

Comment:
I would try to parse the error and see if it's an `AUTH_NOT_AUTHORIZED` error code. We shouldn't be logging out if the API server is unreachable or if an unexpected exception occurs, for example.

Reply:
will add a fix!